devicetypes/drzwave/ezmultipli.src/ezmultipli.groovy

```diff
@@ -49,357 +51,364 @@ metadata {
-       input("lum", "enum", title:"Illuminance Measurement", description: "Percent or Lux", defaultValue: 2 ,required: false, displayDuringSetup: true, options:
-          [1:"Percent",
-           2:"Lux"])
-  }
-
-} // end metadata
+		section {
+			input title: "Reporting Frequencies", description: "Enter values below to tune the reporting frequencies of the sensors in minutes.", displayDuringSetup: false, type: "paragraph", element: "paragraph"
+			input "OnTime",  "number", title: "No Motion Interval", description: "N minutes lights stay on after no motion detected [0, 1-127]", range: "0..127", defaultValue: 2, displayDuringSetup: true, required: false
+			input "LiteMin", "number", title: "Luminance Report Frequency", description: "Luminance report sent every N minutes [0-127]", range: "0..127", defaultValue: 6, displayDuringSetup: true, required: false
+			input "TempMin", "number", title: "Temperature Report Frequency", description: "Temperature report sent every N minutes [0-127]", range: "0..127", defaultValue: 6, displayDuringSetup: true, required: false
+		}
+		section {
+			input title: "Temperature Offset", description: "This feature allows you to correct any temperature variations by selecting an offset. Ex: If your sensor consistently reports a temp that's 5 degrees too warm, you'd enter '-5'. If 3 degrees too cold, enter '+3'.", displayDuringSetup: true, type: "paragraph", element: "paragraph"
+			input "TempAdj", "number", title: "Degrees", description: "Adjust temperature up/down N tenths of a degree F [(-127)-(+128)]", range: "-127..128", defaultValue: 0, displayDuringSetup: true, required: false
+		}
+		section {
+			input title: "Associated Devices", description: "The below preferences control associated node 2 devices.", displayDuringSetup: false, type: "paragraph", element: "paragraph"
+			input "OnLevel", "number", title: "Dimmer Onlevel", description: "Dimmer OnLevel for associated node 2 lights [-1, 0, 1-99]", range: "-1..99", defaultValue: -1, displayDuringSetup: true, required: false
+		}
+	}
+}
+
+private getRED() { "red" }
+private getGREEN() { "green" }
+private getBLUE() { "blue" }
+private getRGB_NAMES() { [RED, GREEN, BLUE] }
+
+def setupHealthCheck() {
+	def motionInterval = (OnTime != null ? OnTime : 2) as int
+	def luminanceInterval = (LiteMin != null ? LiteMin : 6) as int
+	def temperatureInterval = (TempMin != null ? TempMin : 6) as int
+	def interval = Math.max(motionInterval, Math.max(luminanceInterval, temperatureInterval))
+
+	// Device-Watch simply pings if no device events received for twice the maximum configured reporting interval + 2 minutes
+	sendEvent(name: "checkInterval", value: 2 * interval * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+}
+
+def installed() {
+	state.colorReceived = [red: null, green: null, blue: null]
+	setupHealthCheck()
+}
+
+def updated() {
+	log.debug "updated() is being called"
+	def cmds = configure()
```

Comment:
worried about `configure()` getting called twice during install.

Reply:
It will likely on Classic and I believe only does on the new app if the name is changed. Unless I've forgot some of the ways we managed this possibility (I had started on something today that could manage only sending changed configs, but it is starting to get complex), we're probably fine with the risk. The design of the DTH came from the original author, who was adamant about some of it. I can explain in person.